elastic/kibana
# Adds read-only view to role mappings management

Resolves #141802

## Summary

Adds read-only role mappings UI.

### ToDo

- [x] Replace deprecated EUI components (EuiPageContent -> EuiPageSection)
- [ ] Remove code editor read-only tooltip (only appears with delete keypress) - Does not appear possible yet, this feature does not exist in the react code editor component

## Testing

1. Create a user with `read_security` privileges and the `viewer` role (Kibana admin role also works)
2. Login as that user and navigate to Stack Management > Role Mappings
3. There should be a message stating that there are no role mappings, but no button to create one
4. Log in as a superuser to create role mappings that can be viewed
5. Log back in as the test user and verify a read-only view if the created role mappings
6. Lather, rinse, repeat to check various UI modes (roles vs role templates, rule visualization vs raw json)
